### ISSUE TITLE
Распаковка файлов

### ISSUE THREAD

**Issue by FakelsHub** [falltergeist/datfile#5]
Утилита я так понимаю распаковывает все файлы находяшиеся в .dat ?
т.е. она не работает как dat2 которой можно указать какие файлы нужно распаковать.

**Comment by alexeevdv** [comment]
да, с отдельными файлами не работает. я запилил ttvfs_dat2 модуль и на подходе ttvfs_dat1. Это позволит произвольно читать (когда-нибудь и записывать) любой файл в dat архиве. Я плохо представляю как распаковывать отдельные файлы. Передавать их имена по одному? Можешь скинуть пример того как ты видишь запуск консольной команды для распаковки отдельных файлов?

**Comment by FakelsHub** [comment]
Ну дат2 же распаковывает отдельные файлы, там даже можно целый список составить.
Есть же исходники дат2 смотрите изучайте как он распаковывает.
Вообще я почему спрашиваю мне нужен распаковщик в виде длл чтоб я подключил ее к своим утилитам, и избавился от этой консольной дат2. Сами исходники дат2 не собираются  там какаято проблема с zlib, поэтому ищу готовые чтобы переделать под длл версию.(если осилю)

**Comment by alexeevdv** [comment]
возьми ttvfs с моим модулем. можно dll скомпилить или статически слинковать. там то что тебе нужно

**Comment by alexeevdv** [comment]
сегодня вечером обновлю доку чтобы было понятно как этот модуль использовать

**Comment by FakelsHub** [comment]
А не подскажите в чем проблема в сборки dat2, вот лог с ошибкой zlib
```
Создание кода...
zlib.lib(inftrees.obj) : error LNK2001: неразрешенный внешний символ "___report_rangecheckfailure"
D:\My Documents\Visual Studio 2010\Projects\dat2_src_2.32\src\Release\dat2.exe : fatal error LNK1120: 1 неразрешенных внешних элементов
```
Сами [исходники dat2](https://github.com/FakelsHub/Fallout_Dat2) идут почему-то без необходимых zlib файлов, какие нашел в интернете засунул их в проект dat2, но при сборки получал ошибки.
Я пробовал разные zlib'ы  и новые и древние версии ни с какими не собирается эта консольная падла.
